lib/core/src/Cardano/Pool/DB/Sqlite/TH.hs

```diff
@@ -89,4 +93,26 @@ PoolRegistration sql=pool_registration
 
     Primary poolRegistrationPoolId
     deriving Show Generic
+
+-- Temporary queue where metadata to fetch are stored.
+PoolMetadataQueue sql=pool_metadata_queue
+    poolMetadataQueuePoolId            W.PoolId           sql=pool_id
+    poolMetadataQueueUrl               Text               sql=metadata_url
+    poolMetadataQueueHash              ByteString         sql=metadata_hash
+
+    Primary poolMetadataQueuePoolId
+    deriving Show Generic
+
+-- Cached metadata after they've been fetched from a remote server.
+PoolMetadata sql=pool_metadata
```

Comment:
`StakePoolMetadata` contains a field `pledgeAddress :: Text`. Should this type too?

Reply:
We don't have any `PledgeAddress` with cardano-node, but we do have a pledge amount. I've started to rework a bit the metadata types because it's sort of hard to follow with both jormungandr and cardano-node.